src/test/java/pages/HomeGBH.java

```diff
@@ -0,0 +1,96 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class HomeGBH {
+    WebDriver driver;
+
+    @FindBy(xpath = "/html/body/div[2]/div[3]/div[1]/h2")
+    WebElement titleText;
+
+    @FindBy(xpath = "/html/body/header/div[1]/nav/ul[2]/li[1]")
+    WebElement menItemHome;
+
+    @FindBy(xpath = "/html/body/header/div[1]/nav/ul[2]/li[2]")
```

Comment:
Xpaths should be avoided whenever possible, the best way to find an element is by id if it has it as it will find it in case the structure of the page changes.